ohttp/src/config.rs

```diff
@@ -78,6 +79,29 @@ impl KeyConfig {
         })
     }
 
+    /// Construct a configuration from an existing private key
+    /// # Panics
+    /// If the configurations don't include a supported configuration.
+    pub fn import_p384(
```

Comment:
This is a bit tricky. We may have to support a generic version of the import function.